src/main/java/net/floodlightcontroller/util/MatchUtils.java

```diff
@@ -136,6 +136,7 @@
 	public static final String STR_PBB_UCA = "pbb_uca";
 
 	public static final String STR_TCP_FLAGS = "tcp_flags";
+	public static final String STR_OVS_TCP_FLAGS = "ovs_tcp_flags";//experimenter OXM before floodlight completely support of1.5
```

Comment:
@allewwaly to be clear, Floodlight does support OpenFlow 1.5. It's OVS that experimentally supports OpenFlow 1.5